ubi/src/installer.rs

```diff
@@ -394,12 +456,13 @@ impl ArchiveInstaller {
                 | Extension::Tgz
                 | Extension::Txz,
             ) => Self::extract_entire_tarball(downloaded_file, td.path())?,
+            Some(Extension::SevenZip) => Self::extract_entire_7z(downloaded_file, td.path())?,
             Some(Extension::Zip) => Self::extract_entire_zip(downloaded_file, td.path())?,
             _ => {
                 return Err(anyhow!(
                     concat!(
                         "the downloaded release asset, {}, does not appear to be an",
-                        " archive file so we cannopt extract all of its contents",
+                        " archive file so we cannot extract all of its contents",
```

Comment:
I just fixed this in `master` so this change will go away once you merge/rebase.